src/icons/CrossChainIndicator.tsx

```diff
@@ -0,0 +1,31 @@
+import * as React from 'react'
+import colors, { Colors } from 'src/styles/colors'
+import Svg, { G, Circle, Path, Defs, ClipPath } from 'react-native-svg'
+
+interface Props {
+  size?: number
+  color?: colors
+}
+
+function CrossChainIndicator({ size = 16, color = Colors.gray3 }: Props) {
+  return (
+    <Svg width={size} height={size} fill="none">
```

Comment:
i suggest to add `viewBox="0 0 16 16"` so we don't encounter scaling surprises when different size is provided